src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.DecryptToken.cs

```diff
@@ -97,14 +88,16 @@ internal TokenDecryptionResult DecryptToken(
                 var key = ResolveTokenDecryptionKey(jwtToken.EncodedToken, jwtToken, validationParameters, callContext);
                 if (key != null)
                 {
-                    if (LogHelper.IsEnabled(EventLogLevel.Informational))
-                        LogHelper.LogInformation(TokenLogMessages.IDX10904, key);
+                    // TODO: Move to CallContext
```

Comment:
no logic in the `if` now...remove?

Reply:
Commented the rest of the `if` statement, to be removed in a coming PR.